sources/engine/Stride.Games/GameTime.cs

```diff
@@ -94,10 +97,44 @@ public GameTime(TimeSpan totalTime, TimeSpan elapsedTime)
         /// <value><c>true</c> if the <see cref="FramePerSecond"/> and <see cref="TimePerFrame"/> were updated for this frame; otherwise, <c>false</c>.</value>
         public bool FramePerSecondUpdated { get; private set; }
 
+
+
+        /// <summary>
+        /// Gets the amount of time elapsed multiplied by the time factor.
+        /// </summary>
+        /// <value>The warped elapsed time</value>
+        public TimeSpan WarpElapsed 
+        {
+            // TODO: When switching with .NET 5, use the multiply operator instead of this. elapsedGameTime * Factor
+            get => TimeSpan.FromSeconds(Elapsed.TotalSeconds * Factor);
+        }
+
+
+        /// <summary>
+        /// Gets or sets the time factor for the time factor.
+        /// </summary>
+        /// <value>The multiply factor, a double between 0 and 1.</value>
+        public static double Factor
+        {
+            get => factor; 
+            set 
+            {
+                // TODO: Use pattern matching for a clearer code
+                if (value > 1)
+                    factor = 1;
```

Comment:
why clamping the value between 0 and 1 ? i think it just needs to be 0 or higher, no need for preventing it from being higher than 1 cuz someone might set it to 2 (or more) for a fast forward effect